packages/playground/default-svelte-template/vite.config.js

```diff
@@ -1,4 +1,4 @@
-const svelte = require('@sveltejs/vite-plugin-svelte');
+const svelte = require('@sveltejs/vite-plugin-svelte').default;
```

Comment:
it has occurred to me that since we say "if you must use cjs" and then use it in all of our examples, they're not great examples... :)

Reply:
The thing is, these are not only examples but also e2e testbeds. and jest still isn't playing ball even with 27 and initial esm support. Gotta spend more time to convert the whole setup and stuff like https://github.com/facebook/jest/issues/11438 makes me hesitant to even start on it